packages/ui/src/routes/send/SendPage.tsx

```diff
@@ -40,15 +43,33 @@ const SendPage = () => {
     const fromAssetPage = defaultAsset ?? false
     const currentAccount = useSelectedAccount()
 
+    const addressBook = useAddressBook()
+    const addressBookAccounts = useAddressBookAccounts()
+
     // State
     const [selectedAccount, setSelectedAccount] = useState<AccountResult>()
     const [searchString, setSearchString] = useState<string>("")
     const [warning, setWarning] = useState<string>("")
     const [preSelectedAsset, setPreSelectedAsset] = useState<TokenWithBalance>()
     const [isAddress, setIsAddress] = useState<boolean>(false)
 
+    const [addContact, setAddContact] = useState(false)
+
     const searchInputRef = useRef<HTMLInputElement>(null)
 
+    const displayAddToContacts = () => {
```

Comment:
No need for this to be a function (You're calling it twice in the render). Also, there is no need to check whether it is an address or not since you're already comparing to addresses

```javascript
const displayToContacts = !(addressBookAccounts || []).some(({address}) => address.toLowerCase() === searchString.toLowerCase());
```

Reply:
I Agree. I changed it now to an expression. However, we have to check that it is a valid address as we don't want the add to contacts checkbox to appear with invalid input.